Mathlib/Order/Max.lean

```diff
@@ -200,6 +200,10 @@ protected theorem IsBot.isMin (h : IsBot a) : IsMin a := fun b _ => h b
 
 protected theorem IsTop.isMax (h : IsTop a) : IsMax a := fun b _ => h b
 
+theorem IsTop.isMax_iff {α} [PartialOrder α] {i j : α} (h : IsTop i) : IsMax j ↔ j = i := by
```

Comment:
Could you please add the dual statement as well, for `IsBot` and `IsMin`?

Reply:
Done, thanks!